frontend/components/SetupOnRamp/PayingReceivingTable/PayingReceivingTable.tsx

```diff
@@ -221,9 +229,14 @@ export const PayingReceivingTable = ({ onRampChainId }: PaymentTableProps) => {
             )}
           </>
         ),
-        receiving: tokensRequired ? (
-          <ViewReceivingTokens receivingTokens={tokensRequired} />
-        ) : null,
+        receiving: (() => {
+          // For deposit flow: show all tokens from bridge requirements (receivingTokens)
+          // For setup flow: show tokensRequired
+          const tokensToDisplay = tokensRequired || receivingTokens;
+          return tokensToDisplay && tokensToDisplay.length > 0 ? (
+            <ViewReceivingTokens receivingTokens={tokensToDisplay} />
+          ) : null;
```

Comment:
can we check for `isDepositFlow` instead of the comment..
```suggestion
          const tokensToDisplay = isDepositFlow ?  eceivingTokens : tokensRequired;
          return tokensToDisplay && tokensToDisplay.length > 0 ? (
            <ViewReceivingTokens receivingTokens={tokensToDisplay} />
          ) : null;
```